HiP-DataStore/Core/WriteModel/RatingIndex.cs

```diff
@@ -89,6 +93,15 @@ public void AddRating(string userId, byte rate)
             }
         }
 
+        public Dictionary<int,int> GetRatingTable()
+        {
+            Dictionary <int,int> table = new Dictionary <int,int> { {1,0}, {2, 0}, {3, 0}, {4, 0}, {5, 0} };
```

Comment:
You could write this more elegantly using LINQ:
```CSharp
public Dictionary<int,int> GetRatingTable() => _allRates
    .GroupBy(rate => rate.Value)
    .ToDictionary(g => (int)g.Key, g => g.Count());
```

But of course, your solution also works 😉